lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.0.0-beta.13",
+  "lerna": "2.0.0-beta.20",
```

Comment:
Had to upgrade lerna to support all the different ways of specifying versions.

`npm run bootstrap` gets much faster as a result